### ISSUE TITLE
Project Proposal: Space Launch Dictionary 

### ISSUE THREAD

**Issue by robftg** [ebeshero/DHClass-Hub#745]
Seeing as the number of entities that are able to launch equipment into space grows with the expansion of private companies such as SpaceX and Blue Origin there is a ton of data on what could be going up. The Fun part of watching launches is the big firestick that shoots the cargo up. Personally I would use The [Space Launch Report Website](http://www.spacelaunchreport.com/index.html) to look at recent launches and see what vehicles were being used by all agencies NASA, Russian Federation, and Private Companies.
The Project would be compiling each launch into a few details, such as what launch vehicle was used, what engines were used on it, were there any extra boosters, was the launch a success or failure, what cargo was being carried, was it government or private. Then to compile the information into graphics would be showing what vehicles are most popular or most reliable and who uses what vehicles. 
With this being a very technical project I'll also link [This](https://www.youtube.com/watch?v=4QXZ2RzN_Oo) Video by Scott Manley which explains rocket plumbing as well as touches on fuels used. I can recommended his channel as well as he covers a lot of basics on rocket engines, launches, and telemetry. 
**TL;DR** The overall goal would be to make a XML listing of all publicly known (and it's hard to launch a rocket in secret) space launches and to be able to show what is being launched into the space around our little rock in space.

**Comment by bobbyfunks** [comment]
I think this would be interesting.  There's good information that could be marked up.  There is a lot you could do with this categorizing the Russian, NASA and private companies and comparing the three.

**Comment by ghbondar** [comment]
This sounds interesting, but more details would help. First of all, define "recent" ... certainly, orbital launches date back to 1957, if you were to be all-inclusive, but that might be a bit daunting for a class project. Perhaps pick an arbitrary start date, such as the 2011 and the end of the Space Shuttle era. Now, what do you propose to show? This website is an interesting source of data that I didn't know about, and it does seem to be conducive to marking-up. Perhaps, aim to use timeline graphics? Also, I can imagine a pull-down menu being one way to navigate through these data. While details about the specific upper stages and engines, etc., would be interesting, I suspect that they should be saved for later development... I think having an intuitively-navigable database of these launches is a good place to start, in such a format that it can be easily expanded in the future, both to include more past launches, but also ones in the future, to infinity and beyond!

**Comment by robftg** [comment]
My original idea would be working backwards from today to catalog the recent development of private companies and other nations launching micro satellites. My theory is a menu to sort by agency and with at the minimum showing number of launches by vehicle, cargo, and engines used.

**Comment by Bryant-LettucePrime** [comment]
This reminds me a bit of those super depressing maps that show all the nuclear weapons testing done in Nevada/Siberia in the 60's - 80's, only with a more positive spin that the world could _REALLY_ use right now.

**Comment by ebeshero** [comment]
Space Launch is re-launching! Huzzah!
